layers/+lang/elixir/packages.el

```diff
@@ -131,7 +131,9 @@
       "oi" 'alchemist-macroexpand-once-region
       "oI" 'alchemist-macroexpand-once-print-region
       "or" 'alchemist-macroexpand-region
-      "oR" 'alchemist-macroexpand-print-region)
+      "oR" 'alchemist-macroexpand-print-region
+
+      "=" 'elixir-format)
```

Comment:
I think it should be put in `init-elixir-mode` function instead if `elixir-format` if defined by this package.